README.md

```diff
@@ -6,19 +6,19 @@ This is the repository for the open source version of Event Store, which include
 
 ## Support
 
-Information on commercial support and options such as LDAP authentication can be found on the Event Store website at https://eventstore.org/support.
+Information on support and commercial tools such as LDAP authentication can be found here: [Event Store Support]https://eventstore.com/support/.
```

Comment:
Just noticed you are missing the brackets around the link

Reply:
Oh thanks - amended